docs/configuring.rst

```diff
@@ -0,0 +1,60 @@
+================
+Configuration
+================
+
+There are tools to configure some of kopf functionality, like asynchronous
+tasks behaviour and logging events.
+
+.. note::
+    All configuration should be done before importing kopf modules.
+
+
+Configure logging events
+=================
+
+`kopf.config.EventsConfig` allows to set what types of kopf logs should be
+reflected in events.
+
+Loglevels are:
+
+* ``kopf.config.LOGLEVEL_INFO``
+* ``kopf.config.LOGLEVEL_WARNING``
+* ``kopf.config.LOGLEVEL_ERROR``
+* ``kopf.config.LOGLEVEL_CRITICAL``
+
+.. code-block:: python
+    :caption: test_example_operator.py
+
+    from kopf import config
+
+    # Now kopf will send events only when error or critical occasion happens
+    config.EventsConfig.events_loglevel = config.LOGLEVEL_ERROR
```

Comment:
Everything beside thread pool sizes, such as this loglevel, can be set at any time after the imports. The values are used as they are at the moment of their usage, they are never remembered locally. Or do I miss something?

Reply:
You are right, fixed that.